README.md

```diff
@@ -44,6 +46,16 @@ By default both the top-level project and its vendored dependencies are examined
 $ backvendor -deps=false -importpath github.com/example/name src
 ```
 
+If there are additional local files not expected to be part of the upstream version they can be excluded:
+```
+$ cat exclusions
+.git
+Dockerfile
+$ ls src/Dockerfile src/.git
```

Comment:
This looks faked as I'd expect .git to be a directory and ls(1) to need a `-d`?

Reply:
Caught me.